Tools-Override/resolveContract.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you please put an unmodified version of this in an earlier commit so we can see what changed?